scripts/upload-release.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
This should probably go in the *bin* directory since it's meant to be run by a human rather than indirectly via `meson`.